content/klc/G2.1.adoc

```diff
@@ -1,21 +1,47 @@
 +++
-brief = "Defining generic and atomic parts"
+brief = "Generic, fully specified and atomic parts."
 +++
 
-In general terms, the _symbols_ in the KiCad library can be categorized into two groups:
+KiCad is flexible with how symbols and footprints can be connected. In general there are at least two workflows:
+. Generic symbol: Assign the footprint after (or during) symbol placement in eeschema.
+.. Footprint filters may be used to reduce the set of possible footprints.
+. Fully specified symbol: Assign a default footprint in the symbol.
+.. Footprint field is set for one footprint
+.. Footprint filters allow selection of slight variations of this footprint
+
+The later workflow can be split up further
```

Comment:
I think it should be the latter (two t) .... but I think it would be better to actually name it, e.g. "The fully specified symbol workflow ...